### PR TITLE
Implement InfluxDB BackendSubscriber

### PR DESCRIPTION
**Pull request by ChrisBr**
to send performance metrics of the Backend to our InfluxDB.

### CODE REVIEW

File: src/api/config/initializers/influxdb_custom_subscriber.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+if CONFIG['influxdb_hosts'].present?
+  require Rails.root.join('lib', 'influxdb_obs', 'influxdb_obs').to_s
```
Comment (hennevogel) [inline src/api/config/initializers/influxdb_custom_subscriber.rb]: All files in `lib/` are autoloaded...

Comment (ChrisBr) [reply src/api/config/initializers/influxdb_custom_subscriber.rb]: Ah no? All code in app is autoloaded, lib is not autoloaded and either needs to be specified in application.rb or in the initializer.

https://stackoverflow.com/a/40019108

**Comment by ChrisBr** [comment]
@hennevogel updated

File: src/api/config/initializers/influxdb_custom_subscriber.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+require_relative File.join('..', '..', 'lib', 'influxdb_obs', 'influxdb_obs')
```
Comment (hennevogel) [inline src/api/config/initializers/influxdb_custom_subscriber.rb]: That one eats strings....

File: src/api/lib/backend/backend.rb [hunk 3]
```diff
@@ -12,6 +12,7 @@ module Api
 
 require 'benchmark'
 require 'api_exception'
+require_relative File.join('..', 'influxdb_obs', 'obs', 'normalizer', 'location_normalizer')
```
Comment (hennevogel) [inline src/api/lib/backend/backend.rb]: Why do we requires this here and not in the logger?

**Comment by ChrisBr** [comment]
coveralls hanging ...

**Pull request merged**